js-rattler/src/Version.ts

```diff
@@ -195,4 +195,13 @@ export class Version {
     public withAlpha(): Version {
         return Version.fromNative(this.native.with_alpha());
     }
+
+    /**
+     * Compare two versions.
+     *
+     * Returns `-1` if this<other, `0` if this == other, `1` if this > other
```

Comment:
It's a nitpick but it would be nice if `this < other`, ... would have the same spacing formatting as the others.